Base/usr/share/man/man2/mount.md

```diff
@@ -37,14 +39,22 @@ mounted file system.
 
 ### Bind mounts
 
-If `MS_BIND` is specified in `flags`, `fs_type` is ignored and a bind mount is performed
-instead. In this case `source` is treated as a path to a file or directory whose contents
-are overlayed over `target`. This can be used  as an alternative to symlinks or hardlinks.
+If `MS_BIND` is specified in `flags`, `fs_type` is ignored and a bind mount is
+performed instead. In this case, the file or directory specified by `source_fd`
+is overlayed over `target` — the target appears to be replaced by a copy of the
+source. This can be used as an alternative to symlinks or hardlinks.
 
 ## Errors
 
+* `EFAULT`: The `fs_type` or `target` are invalid strings.
 * `EPERM`: The current process does not have superuser privileges.
-* `ENODEV`: The `fs_type` is unrecognized, or the device is not found, or the device doesn't contain a valid filesystem image.
+* `ENODEV`: The `fs_type` is unrecognized, or the file descriptor to source is
+  not found, or the source doesn't contain a valid filesystem image. Also, this
+  error occurs if `fs_type` is valid, but the file descriptor from `source_fd`
+  is not seekable.
+* `EBADF`: If the `source_fd` is not valid, and either `fs_type` specifies a
+  file-backed filesystem (and not a pseudo filesystem), or `MS_BIND` is
+  specified in flags.
```

Comment:
This doesn't render correctly in the `man` or `Help` programs. Seems like we don't support newlines in the middle of bullet point lists.